src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -868,6 +868,11 @@
         },
         "BGP_NEIGHBOR": {
             "default|192.168.1.1": {
+                "asn": "65200",
+                "holdtime": "180",
```

Comment:
"BGP_NEIGHBOR": {
            "default|192.168.1.1": {
                "asn": "65200",
                "holdtime": "180",
                "keepalive": "60",
                "local_addr": "10.0.0.2",
                "name":"PEER2"
            }
        },

@praveen-li These fields are present in the grouping, looks like libyang is not able to parse it.

 Read JSON Section: SAMPLE_CONFIG_DB_JSON
sonic_yang(3):All Keys are not parsed in BGP_NEIGHBOR
dict_keys(['default|192.168.1.1'])
sonic_yang(3):exceptionList:['Value not found for neighbor in default|192.168.1.1', "'asn'"]
sonic_yang(3):Data Loading Failed:All Keys are not parsed in BGP_NEIGHBOR
dict_keys(['default|192.168.1.1'])

Reply:
@venkatmahalingam:  Yeah, [current translation doesn't allow all the variation of leaf, only choice and leaf is allowed.] if you are using grouping, please add support at:
https://github.com/Azure/sonic-buildimage/blob/5acf2348ba88c69bad3300adabd58b270e5bc01f/src/sonic-yang-mgmt/sonic_yang_ext.py#L209 

Also, about the other BGP_NEIGHBOR table
 -- where the relevant config go, as I understand BGP_NEIGHBOR table in config_db is different.
 -- which daemon processes it, does it make use of current YANG models. 
 -- please point to the code, which is using this BGP_NEIGHBOR yang model.

Reply:
New generic BGP config models are being processed by the below frrcfgd daemon in BGP container.
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-frr-mgmt-framework/frrcfgd/frrcfgd.py

Reply:
@venkatmahalingam: can you please point to the function or relevant line in frrcfgd.py, where it is doing config validation against yang. Also from where it reads the config i.e. source of config?.

Reply:
@praveen-li There is no config validation against YANG in frrcfgd, frrcfgd simply listens for config-DB events and handles them, all configs validations are expected to handle in SONiC YANGs before pushing to config-DB.